docs/etcd_clusters.md

```diff
@@ -0,0 +1,60 @@
+# Manage etcd clusters
+
+Mayu contains an etcd discovery to setup and manage your etcd clusters. During startup you can configure if you like to use an external etcd discovery or Mayu itself.
```

Comment:
ok, thats what I meant. I would then suggest to replace `--etcd-endpoint=http://localhost:2379` with something like `--use-mayu-etcd-discovery` as the port referenced here might change etc and is more explicit.